neighbour-model/src/test/java/no/vegvesen/ixn/federation/model/NeighbourTest.java

```diff
@@ -119,34 +119,6 @@ public void shouldCheckSubscriptionRequestsForUpdates_neighbourWithNewCapability
 		assertThat(neighbourWithNewerCapabilitiesThanSubscriptionRequest.shouldCheckSubscriptionRequestsForUpdates(Optional.of(localSubscriptionUpdateTimeBeforeCapabilityPost))).isTrue();
 	}
 
-	@Test
-	public void failedSubscriptionRequest_firstSetsStart() {
-		Neighbour neighbour = new Neighbour("the-best-neighbour-ever", null, null, null);
-		neighbour.failedConnection(2);
-		assertThat(neighbour.getBackoffStartTime()).isNotNull().isAfter(LocalDateTime.now().minusSeconds(3));
-		assertThat(neighbour.getBackoffAttempts()).isEqualTo(0);
-		assertThat(neighbour.getConnectionStatus()).isEqualTo(ConnectionStatus.FAILED);
-
-		neighbour.failedConnection(2);
-		assertThat(neighbour.getBackoffAttempts()).isEqualTo(1);
-		assertThat(neighbour.getConnectionStatus()).isEqualTo(ConnectionStatus.FAILED);
-
-		neighbour.failedConnection(2);
-		assertThat(neighbour.getBackoffAttempts()).isEqualTo(2);
-		assertThat(neighbour.getConnectionStatus()).isEqualTo(ConnectionStatus.FAILED);
-
-		neighbour.failedConnection(2);
-		assertThat(neighbour.getConnectionStatus()).isEqualTo(ConnectionStatus.UNREACHABLE);
-
-		neighbour.failedConnection(2);
-		assertThat(neighbour.getConnectionStatus()).isEqualTo(ConnectionStatus.UNREACHABLE);
-
-		neighbour.okConnection();
-		assertThat(neighbour.getConnectionStatus()).isEqualTo(ConnectionStatus.CONNECTED);
-		assertThat(neighbour.getBackoffStartTime()).isNull();
-		assertThat(neighbour.getBackoffAttempts()).isEqualTo(0);
-	}
-
 	@Test
 	public void calculatedNextPostAttemptTimeIsInCorrectInterval(){
```

Comment:
Take out the reference to Neighbour, move the test to ConnectionBackoffTest, working directly on the ConnectionBakoff class

Reply:
Done :)